### ISSUE TITLE
Use modern constructor syntax

### ISSUE THREAD

**Issue by asmecher** [pkp/pkp-lib#1958]
PHP7 will soon drop support for old-style `ClassName()` constructors; use `function __construct` instead.

**Comment by asmecher** [comment]
This is all merged (and should fix the tests, @NateWr and @bozana, for OJS master). I'll port to OMP master when I get the chance (probably tomorrow).

**Comment by bozana** [comment]
Great work @asmecher! Thanks a lot!

**Comment by NateWr** [comment]
I ran into an issue that I think is related to this. The Site Settings form calls `parent::Form()` directly in the constructor:

https://github.com/pkp/pkp-lib/blob/master/controllers/tab/settings/siteSetup/form/SiteSetupForm.inc.php#L26

This method no longer exists. It seems like a reference to the `Form` class's constructor, which I suppose was used to bypass calling the constructor of the direct parent class, `PKPSiteSettingsForm`, as indicated by the comment above that line.

Just switching it over to calling the parent constructor doesn't work. It embeds an entire HTML element when it is supposed to load just the HTML to inject into a tab.

I did a quick search and noticed this pattern of calling `parent::Form` was also used by some forms in the subscriptions plugin.
